src/private/warehouse.ts

```diff
@@ -67,6 +152,7 @@ export const warehousePrivateHandlers: PrivateNatsHandler[] = [
                 const { start, limit } = (data as unknown) as SearchQuery;
 
                 try {
+                    await ItemSpecsSchema.validate({ start, limit });
```

Comment:
I don't think this should be called "ItemSpecsSchema", this is more like "GetItemsRequest"